src/components/search/index.tsx

```diff
@@ -37,12 +45,15 @@ export const SearchInput: React.FC = () => {
   }, []);
```

Comment:
Is this working okay if you type something and then something else? Just asking because [] works on the first render. Here, we want it to work when the searchTerm changes as well. We want another API request. So you need to pass searchTerm as a dependency to the array. And now that I'm thinking about that, it's probably best to have searchTerm as a piece of state so you can use it as a dependency

Reply:
yeah it works when you type something and then delete it and type it again and if you use mulitple words